posts/for-vs-apply/index.qmd

```diff
@@ -0,0 +1,148 @@
+---
+title: "Why use `apply()` instead of `for` loops?"
+subtitle: "Going beyond the debunked performance argument."
```

Comment:
This was a really concise read and conveyed the message quite quickly. 

My only comment is that the current title might undersell the article as many people might have read articles with a similar title but different content. I would entitle it as "Beyond the need for speed: Lesser-known reasons to prefer `apply()` functions over `for` loops" or something along those lines to draw out the content of the article. I was trying to play on words with ["need for speed"](https://en.wikipedia.org/wiki/Need_for_Speed).